cordova/config.xml

```diff
@@ -1,88 +1,100 @@
-    <icon src="res/ios/Icon-60x60@3x.png" width="180" height="180"/>
-    <icon src="res/ios/Icon-72x72@2x~ipad.png" width="144" height="144"/>
-    <icon src="res/ios/Icon-72x72~ipad.png" width="72" height="72"/>
-    <icon src="res/ios/Icon-76x76@2x~ipad.png" width="152" height="152"/>
-    <icon src="res/ios/Icon-76x76~ipad.png" width="76" height="76"/>
-    <icon src="res/ios/icon29x29.png" width="29" height="29"/>
-    <splash src="res/ios/Default~iphone.png" width="320" height="480"/>
-    <splash src="res/ios/Default@2x~iphone.png" width="640" height="960"/>
-    <splash src="res/ios/Default-568h@2x~iphone.png" width="640" height="1136"/>
-    <splash src="res/ios/Default-667h.png" width="750" height="1334"/>
-    <splash src="res/ios/Default-736h.png" width="1242" height="2208"/>
-    <splash src="res/ios/Default-1024h.png" width="768" height="1024"/>
-    <splash src="res/ios/Default-2048h.png" width="1536" height="2048"/>
-    <splash src="res/ios/Default-768h.png" width="1024" height="768"/>
-    <splash src="res/ios/Default-1536h.png" width="2048" height="1536"/>
-    <splash src="res/ios/Default-1242h.png" width="2208" height="1242"/>
-    <feature name="StatusBar">
-      <param name="ios-package" onload="true" value="CDVStatusBar"/>
-    </feature>
-    <preference name="Orientation" value="portrait"/>
-    <preference name="Fullscreen" value="true"/>
-  </platform>
-  <access origin="tel:*" launch-external="yes"/>
-  <access origin="*"/>
-  <allow-intent href="https://maps.google.com/*"/>
-  <engine name="ios" spec="4.2.0"/>
-  <engine name="android" spec="~5.1.1"/>
-  <engine name="windows" spec="~4.3.1"/>
-  <plugin name="cordova-plugin-statusbar" spec="~2.1.1"/>
-  <plugin name="phonegap-plugin-push" spec="~1.4.5">
-    <variable name="SENDER_ID" value="XXXXXXX"/>
-  </plugin>
-  <plugin name="cordova-plugin-device" spec="^1.1.1"/>
-  <plugin name="cordova-plugin-whitelist" spec="~1.2.1"/>
-  <plugin name="cordova-plugin-inappbrowser" spec="~1.2.0"/>
-  <plugin name="cordova-plugin-network-information" spec="^1.2.0"/>
-  <plugin name="cordova-plugin-actionsheet" spec="https://github.com/EddyVerbruggen/cordova-plugin-actionsheet.git"/>
-  <plugin name="cordova-plugin-camera" spec="~2.1.0"/>
-  <plugin name="cordova-plugin-splashscreen" spec="~3.1.0"/>
-  <plugin name="phonegap-plugin-barcodescanner" spec="https://github.com/wildabeast/BarcodeScanner.git"/>
-  <plugin name="com.phonegap.plugins.twilioclient" spec="https://github.com/crossroads/twilio_client_phonegap.git"/>
-  <plugin name="cordova-plugin-ios-no-export-compliance" spec="https://github.com/mikaoelitiana/cordova-plugin-ios-no-export-compliance.git" />
-  <plugins></plugins>
+<?xml version='1.0' encoding='utf-8'?>
+<widget android-versionCode="0" id="hk.goodcity.adminstaging" ios-CFBundleVersion="0" version="0.14.017" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+    <name>S. Admin GoodCity</name>
```

Comment:
Can we use 2 spaces convention here over 4 space as we've used 2 space convention everywhere @shivanibhanwal ?

Reply:
This I can fix it is my new editor I did not paid attention sorry about that.